script.js

```diff
@@ -0,0 +1,174 @@
+    triggerEl.addEventListener('click', function (event) {
+      event.preventDefault();
+      const tabId = triggerEl.getAttribute('href');
+      showTab(tabId);
+      currentTab = tabId;
+      updateURL(currentTab);
+    })
+  });
+
+  window.addEventListener('popstate', function(event) {
+    currentTab = getCurrentTabFromURL() || '#activity';
+    showTab(currentTab);
+  });
+
+  HideElems();
+  AdaptHeader();
+
+  if (!document.location.hash) {
+    showTab('#activity');
+  }
+
+  function getCurrentTabFromURL() {
+    const hash = document.location.hash;
+    return hash ? hash : null;
+  }
+
+  function showTab(tabId) {
+    const activeTriggerEl = triggerTabList.find(triggerEl => triggerEl.getAttribute('href') === tabId);
+    if (activeTriggerEl) {
+      const tabTrigger = new bootstrap.Tab(activeTriggerEl);
+      tabTrigger.show();
+    }
+  }
+
+  function updateURL(tabId) {
+    history.pushState(null, null, tabId);
+  }
+});
+
+ymaps.ready(init);
+
+var myMap
+function init(){
+  myMap = new ymaps.Map("map_img", {
+    center: [51.83, 107.6],
+    zoom: 8
+  });
+
+  var loader = document.getElementById('loader');
+  loader.style.display = 'none';
+
+  myMap.events.add('tilesloadstart', function () {
+    loader.style.display = 'block';
+  });
+
+  myMap.events.add('tilesloaded', function () {
+    loader.style.display = 'none';
+  });
+
+  var myPlacemark = new ymaps.Placemark([51.83, 107.6], {
+    hintContent: 'Улан-Удэ'
+  });
+  myMap.geoObjects.add(myPlacemark);
+}
+
+function reload_map() {
+  myMap.destroy()
+  init()
+}
+
+function AdaptHeader() {
+  let menuMainHeader = document.querySelector("#menu_main_header .dropdown-menu");
+  let mainNav = document.getElementById("main_nav");
+
+  let menuHeader = document.querySelector('#menu_header .dropdown-menu');
+  let headerNavSettings = document.getElementById("header_nav_settings");
+
+  for (const child of headerNavSettings.children) {
+    const listItem = document.createElement('li');
+    listItem.classList.add('dropdown-item');
+    listItem.appendChild(child.cloneNode(true));
+    menuHeader.appendChild(listItem);
+  }
+
+  for (const child of mainNav.children) {
+    const listItem = document.createElement('li');
+    listItem.classList.add('dropdown-item');
+    listItem.appendChild(child.cloneNode(true));
+    menuMainHeader.appendChild(listItem);
+  }
+}
+
+let startTime
+function start_time() {
+  startTime = Date.now();
+
+  setInterval(() => {
+    const timeElapsed = Math.floor((Date.now() - startTime) / 1000);
+    const hours = Math.floor(timeElapsed / 3600);
+    const minutes = Math.floor((timeElapsed / 60) % 60);
+    const seconds = timeElapsed % 60;
+
+    document.querySelector('#timer').textContent = `${hours.toString().padStart(2, '0')}:${minutes.toString().padStart(2, '0')}:${seconds.toString().padStart(2, '0')}`;
+  }, 1000);
+}
+
+function HideElems() {
+  let dropdown_share_thoughts = document.getElementById("dropdown_share_thoughts")

```

Comment:
у нас js а не питон старайся кейс не менять в программе
```suggestion
  let dropdownShareThoughts = document.getElementById("dropdown_share_thoughts")

```